components/data/data-sources/src/main/java/org/eclipse/dirigible/components/data/sources/service/DataSourceLifecycleListener.java

```diff
@@ -0,0 +1,11 @@
+package org.eclipse.dirigible.components.data.sources.service;
+
+import org.eclipse.dirigible.components.data.sources.domain.DataSource;
+
+public interface DataSourceLifecycleListener {
+
+    void onSaved(DataSource dataSource);
```

Comment:
let use present tense here: onSave() on onDelete()
usually `on` is associated with `after` when there is only single event used

Reply:
done